src/xcode/ENA/ENA/Source/Services/PPAnalyticsCollector/Model/PPADataType.swift

```diff
@@ -49,6 +49,25 @@ enum PPATestResultMetadata {
 }
 
 enum PPAKeySubmissionMetadata {
+	case create(KeySubmissionMetadata, SubmissionTestType)
+	case submitted(Bool, CoronaTestType)
+	case submittedInBackground(Bool, CoronaTestType)
+	case submittedAfterCancel(Bool, CoronaTestType)
+	case submittedAfterSymptomFlow(Bool, CoronaTestType)
```

Comment:
submittedAfterSymptom **s** Flow?

Reply:
`submittedAfterSymptomFlow` is as per techspecs